react/AppIcon/styles.styl

```diff
@@ -0,0 +1,36 @@
+@keyframes placeHolderShimmer {
+  0% { background-position: -20rem 0; }
+  80% { background-position: 20rem 0; }
+  80.1% { background-position: -20rem 0; }
+  100% { background-position: -20rem 0; }
+}
+
+.c-loading-placeholder {
+    animation-duration: 2s;
+    animation-iteration-count: infinite;
+    animation-name: placeHolderShimmer;
+    animation-timing-function: linear;
+    background-position: -20rem 0;
+    background-image: linear-gradient(to right, silver 0%, #fcfcfc 50%, silver 100%);
+    background-size: 20rem 10rem;
+    background-repeat: no-repeat;
+    background-color: silver;
+    border-radius: 0.15rem;
+    max-width: 100%;
+    position: relative;
+    margin: .1rem 0;
+}
+
+.c-app-icon,
+.c-app-icon svg {
+    height 100%
+    width 100%
+}
+
+.c-app-icon-default {
+    color silver
+}
+
+.c-app-icon-placeholder {
+    animation-delay: 0.1s;
```

Comment:
Should this class be merged with `.c-loading-placeholder` ?

Maybe stylus composition features could be used : 

```
.c-app-icon-placeholder {
  @extends .c-loading-placeholder
  animation-delay: 0.1s;
}
```

This way `AppIcon` would only need to apply `.c-app-icon-placeholder` instead `.c-loading-placeholder` and `.c-app-icon-placeholder`.